src/transformers/__init__.py

```diff
@@ -115,9 +115,10 @@
         "load_tf2_model_in_pytorch_model",
         "load_tf2_weights_in_pytorch_model",
     ],
-    "models": [],
     # Models
+    "models": [],
     "models.albert": ["ALBERT_PRETRAINED_CONFIG_ARCHIVE_MAP", "AlbertConfig"],
+    # Models
```

Comment:
Something weird happened here; normally this PR shouldn't change this piece of code.

Reply:
No clue about this one

Reply:
You can just look at how it looks in the main branch, and then update the code here.